src/Facade/Services/IFacadeWithSearchReturnTen.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Linn.Stores.Facade.Services
+{
+    using System.Collections.Generic;
+
+    using Linn.Common.Facade;
+
+    public interface IFacadeWithSearchReturnTen<T, in TKey, in TResource, in TUpdateResource> : IFacadeService<T, TKey, TResource, TUpdateResource>
```

Comment:
Does this need to be so specific? How about a search method with an option limit parameter that you could set to ten, or anything you like. I'd also include the word 'Service' in the interface name somewhere for conventions sake.

Reply:
Yeah I can change it to have an optional limit and chuck service in the name, I basically just copied it in from production to use it in a similar way